pkgs/top-level/all-packages.nix

```diff
@@ -6903,6 +6903,8 @@ with pkgs;
 
   httplab = callPackage ../tools/networking/httplab { };
 
+  hashlink = callPackage ../development/interpreters/hashlink { };
```

Comment:
`hashlink` is not at alphabetical order.
Also add it to the correct section ( `development/interpreters`)